### PR TITLE
Add information on GPU compute modes

### PR DESCRIPTION
**Pull request by stevenvdb**
Adds explanation of the exclusive_process, exclusive_thread, and default features that can be specified for GPUs. Probably only relevant for advanced users.

### CODE REVIEW

**Comment by moravveji** [comment]
Looks good to me. All new features that pass the unittests are explained. Thanks.

File: source/leuven/genius_quick_start.rst [hunk 1]
```diff
@@ -57,14 +57,36 @@ To submit to a compute node it all boils down to specifying the required number
...
+
+   $ qsub -l nodes=1:ppn=4:gpus=1:cascadelake -l partition=gpu  -A myproject  myscript.pbs
+  
+For the V100 type of GPU, it is required that you request 4 cores for each GPU.
+
+Advanced usage
+^^^^^^^^^^^^^^
+There are different GPU compute modes available, which are explained on this `documentation page <http://docs.adaptivecomputing.com/9-1-0/MWM/help.htm#topics/moabWorkloadManager/topics/accelerators/nvidiaGpus.htm>`_.
+
+- default: shared mode available for multiple processes
```
[the remark this answers is not in the document]
Comment (stevenvdb) [reply source/leuven/genius_quick_start.rst:78]: This is specified on line 88. Maybe it is clearer to also move "exclusive process" up in the list of modes.

**Comment by moravveji** [comment]
@stevenvdb: I guess our branches have conflict of interest ;-)
I merged your former branch into the `development` branch, then, I branched off from it, and made a pull request (working on the same file). See https://github.com/hpcleuven/VscDocumentation/pull/177
Now, I see that you have added an extra commit to the old branch.

Now, if both of us merge our branches, the 2nd merge will fail/conflict. To avoid that, I propose you commit first, and I redo my work because it is less textual changes, I presume. Or perhaps, you can also include changes from my branch into yours. Then, we hit two birds with one stone.

What do you thing?

**Comment by stevenvdb** [comment]
I do not see the problem. All commits from this PR are in the development branch (it was merged after the second commit).
In your PR #177 it says there are no conflicts, so that seems fine.

**Pull request merged**